bazel/repositories.bzl

```diff
@@ -465,8 +465,8 @@ def _com_google_absl():
         actual = "@com_google_absl//absl/debugging:symbolize",
     )
     
```

Comment:
run `tools/check_format.py fix`, it is complaining about trailing whitespace on this line.